tools/static-assets/skel/test/main.js

```diff
@@ -0,0 +1,27 @@
+import assert from "assert";
+
+export const testMessage = "Welcome to Meteor!";
+
+describe("~name~", () => {
+  it("package.json has correct name", () => {
```

Comment:
I know it doesn't really matter with these tests, but [arrow function use is discouraged by Mocha](https://mochajs.org/#arrow-functions). We might want to drop them here to set an example.

Reply:
Perhaps use normal `function`s and put a comment under the first one to explain that it's done intentionally to give access to the Mocha API (for things like `this.wait(2000)`, etc.):

```js
// Using `function`s, rather than arrow-functions, allows access to
// Mocha's API.  See https://mochajs.org/#arrow-functions for details.
```

Otherwise the first reaction of many might be:

"Oh, what are these doing here?" => **replace with arrow functions**

Reply:
I'm fine with just using normal functions and defending the choice later, if anyone asks.